source/importing-css-assets-and-javascript/index.html.md.erb

```diff
@@ -72,6 +72,23 @@ If you want to override GOV.UK Frontend's styles with your own styles, `@import`
 
 If your project uses GOV.UK Frontend toolkit, GOV.UK Template or GOV.UK Elements, you can [configure GOV.UK Frontend to work with them](/compatibility-mode/).
 
+### Silence deprecation warnings in Dart Sass
+
+If you are using Dart Sass v1.33.0 or greater, you may see deprecation warnings when compiling your Sass:
+
+```
+DEPRECATION WARNING: Using / for division is deprecated and will be removed in Dart Sass 2.0.0.
+```
+
+We are currently unable to fix these deprecation warnings without breaking support for LibSass.
+
+You can however silence the warnings. Ensure you are using Dart Sass 1.35.0 or greater and then:
```

Comment:
```suggestion
However, you can silence the warnings caused by dependencies, including GOV.UK Frontend. Make sure you're using Dart Sass 1.35.0 or greater, and then:
```

Would also move this content back into the previous paragraph, to make it the next sentence after '..without breaking support for LibSass'.

Reply:
Also, 'greater' or 'higher'?

Reply:
I think they get used interchangeably, but I'd use 'greater' on the basis that version constraints often use the '>' (greater than) symbol, and [the node-semver documentation also uses 'greater than'](https://github.com/npm/node-semver#ranges).

Greater is perhaps a bit more formal, but it also seems more accurate as it's rooted in _value_ whereas higher is rooted in _height_?

Thoughts?